gamla/apply.py

```diff
@@ -0,0 +1,67 @@
+from typing import Text
```

Comment:
maybe we can add the `prepare_and_apply` methods here?

Reply:
it's causing a circular dependency